app/src/test/java/org/simple/clinic/settings/changelanguage/ChangeLanguageUpdateTest.kt

```diff
@@ -0,0 +1,74 @@
+package org.simple.clinic.settings.changelanguage
+
+import com.spotify.mobius.test.NextMatchers.hasEffects
+import com.spotify.mobius.test.NextMatchers.hasModel
+import com.spotify.mobius.test.NextMatchers.hasNoEffects
+import com.spotify.mobius.test.NextMatchers.hasNoModel
+import com.spotify.mobius.test.UpdateSpec
+import com.spotify.mobius.test.UpdateSpec.assertThatNext
+import org.junit.Test
+import org.simple.clinic.settings.Language
+
+class ChangeLanguageUpdateTest {
+
+  private val defaultModel = ChangeLanguageModel.FETCHING_LANGUAGES
+
+  private val englishIndia = Language(displayName = "English", languageCode = "en_IN")
+  private val hindiIndia = Language(displayName = "हिंदी", languageCode = "hi_IN")
+
+  private val spec = UpdateSpec<ChangeLanguageModel, ChangeLanguageEvent, ChangeLanguageEffect>(ChangeLanguageLogic::update)
+
+  @Test
+  fun `when the current selected language is loaded, the user must be notified`() {
```

Comment:
User must be notified? Are we showing a toast / snack bar?

Reply:
Reworded to be less specific.